### ISSUE TITLE
limiter float too generic

### ISSUE THREAD

**Issue by DanielRuf** [magento/magento2#15323]
### Preconditions

1. Magento 2.2.4

```
@media (min-width: 768px), print
.limiter {
    float: right;
}
.products.wrapper ~ .toolbar .pages {
    float: left;
}
```
### Steps to reproduce

1. inspect toolbar in product list
2. compare styles of `.limiter` and `.pages`

### Expected result

1. `.limiter` should have the same parent selectors like `.pages` to prevent clashes between styles and layouts

### Actual result

1. `.limiter` is too generic and is used as single selector for floating the element

**Comment by hitesh-wagento** [comment]
I working on this

**Comment by sidolov** [comment]
Hi @DanielRuf. Thank you for your report. 
The issue has been fixed in magento/magento2#15880 by @hitesh-wagento in 2.1-develop branch
Related commit(s):
 - [950a668544e82b4aeee67dd1beed861c2e195bf5](https://github.com/magento/magento2/commit/950a668544e82b4aeee67dd1beed861c2e195bf5)

The fix will be available with the upcoming 2.1.15 release.

**Comment by magento-engcom-team** [comment]
Hi @DanielRuf. Thank you for your report. 
The issue has been fixed in magento/magento2#15879 by @hitesh-wagento in 2.3-develop branch
Related commit(s):
 - [172e5ae2b2357d6ccd4f848d866b8bee7405d56c](https://github.com/magento/magento2/commit/172e5ae2b2357d6ccd4f848d866b8bee7405d56c)

The fix will be available with the upcoming 2.3.0 release.

**Comment by magento-engcom-team** [comment]
Hi @DanielRuf. Thank you for your report. 
The issue has been fixed in magento/magento2#15878 by @hitesh-wagento in 2.2-develop branch
Related commit(s):
 - [1b9043b64c72e2f1bd83fa517edfc17fa15d2c63](https://github.com/magento/magento2/commit/1b9043b64c72e2f1bd83fa517edfc17fa15d2c63)

The fix will be available with the upcoming 2.2.6 release.
